examples/chef/common/clusters/door-lock/chef-lock-endpoint.cpp

```diff
@@ -590,8 +590,8 @@ bool LockEndpoint::weekDayScheduleForbidsAccess(uint16_t userIndex, bool * haveS
     auto chipError = chip::System::SystemClock().GetClock_RealTimeMS(cTMs);
     if (chipError != CHIP_NO_ERROR)
     {
-        ChipLogError(Zcl, "Lock App: unable to get current time to check user schedules [endpointId=%d,error=%d (%s)]", mEndpointId,
-                     chipError.AsInteger(), chipError.AsString());
```

Comment:
In other places in the codebase we have `static_cast<int>(err.AsInteger())`, so if the error number is valuable here maybe cast will better?

Reply:
@arkq On some platform (e.g. ARM 64 Linux) this could be `unsigned long`. So I don't think it still depends on the platform and we don't have a good way for all platforms. That's why I want to remove this at all.